.travis.yml

```diff
@@ -97,15 +97,15 @@ matrix:
     addons:
       artifacts:
         paths:
-          - .stack-work/dist/x86_64-osx/Cabal-2.2.0.1/build/pact/pact
+          - .stack-work/dist/x86_64-osx/Cabal-2.4.0.1/build/pact/pact
 
   # Linux build
   - env: BUILD=stack ARGS="-j1" Z3VERSION="4.8.3"
     addons:
       apt: {packages: [libgmp-dev]}
       artifacts:
         paths:
-          - .stack-work/dist/x86_64-linux/Cabal-2.2.0.1/build/pact/pact
+          - .stack-work/dist/x86_64-linux/Cabal-2.4.0.1/build/pact/pact
```

Comment:
These paths would change with the bump to `lts-13.*`. Does another tool / integration expect these paths to have an exact shape (say `2.2` vs `2.4`), say for deployment? I wouldn't want to break anything here.

Reply:
Maybe windows? Trying to find out where our windows life is at currently elsewhere too. @vaibhavsagar ?